lib/network/options.js

```diff
@@ -68,6 +68,7 @@ let allOptions = {
       },
       __type__: { string: ["from", "to", "middle"], object }
     },
+    endpointsAdjustedPosition: { number },
```

Comment:
I don't like the name. Ideally you should see the name and immediately know what it does. When I saw this for the first time I thought it overrides the old positioning, only after I read the docs part I understood (I skimmed through the code first, then read the PR text). I suggest something like `endPointOffset`. Of course your free to suggest alternatives or improvements.

Reply:
I think endPointOffset is a good name, thanks.